### ISSUE TITLE
Rcpp version pinned in r_deps list

### ISSUE THREAD

**Issue by sdx23** [mlell/tapas#3]
In the r_deps list the version of Rcpp is pinned to 0.12.15. This prevents installation with "newer" versions of R (such as 3.5.3 :). I'm not too much into R but if I understood correctly one cannot arbitrarily mix Rcpp and R versions.

The following installation procedure worked for me (providing Rcpp via r-essentials):

    git clone https://github.com/mlell/tapas tapas
    cd tapas
    module load Anaconda/3.7
    conda create -n tapas -c bioconda -c r python=3.6 r=3.6 r-essentials
    source activate tapas
    nano scripts/gen/r_deps.txt  # remove the line containing Rcpp
    scripts/gen/install_dependencies
    pip install -r scripts/gen/py_requirements.txt --target scripts/lib/python-ext/
    bash scripts/gen/gen-launchers.sh

**Comment by mlell** [comment]
Yes, this is true. Currently I try to manage the repository to work for the software described in the Singularity recipe which is added to the tree by now. The frozen versions are meant to document the status when the paper on TAPAS was published but this causes problems increasingly. I will update the dependencies, however, not in the coming two weeks for time reasons. Thanks for reporting all the problems!

**Comment by jlapaijmans** [comment]
This solution is working brilliantly for me. If I find a moment, I will add a link to this issue or some such in the readme, in case other people have the same problem and require the same solution!
